src/arch/arch_default.h

```diff
@@ -7,20 +7,18 @@
 #include "LoadingWindow/LoadingWindow_Win32.h"
 #include "LowLevelWindow/LowLevelWindow_Win32.h"
 #include "MemoryCard/MemoryCardDriverThreaded_Windows.h"
-#define DEFAULT_INPUT_DRIVER_LIST "DirectInput,Pump,Para"
-#define DEFAULT_MOVIE_DRIVER_LIST "FFMpeg,Null"
-#define DEFAULT_SOUND_DRIVER_LIST "WaveOut,DirectSound-sw,WDMKS,Null"
-
+const std::vector<RString> DEFAULT_INPUT_DRIVER_LIST = {"DirectInput", "Pump", "Para"};
```

Comment:
Vectors and Strings are not trivially destructible.

If you want to do this then you have to do one of two things:
- you can use an array of char* instead
- create a function that returns a static pointer reference
```cpp
std::vector<RString>& DefaultInputDriverList()
{
  static std::vector<RString>* ret = {"DirectInput", "Pump", "Para"};
  return *ret;
}
```
https://google.github.io/styleguide/cppguide.html#Static_and_Global_Variables